utils/wasm-gen/src/config.rs

```diff
@@ -156,6 +161,7 @@ impl Default for StandardGearWasmConfigsBundle {
     fn default() -> Self {
         Self {
             log_info: Some("StandardGearWasmConfigsBundle".into()),
+            waiting_probability: Some(4),
```

Comment:
I'd reduce it to 8-12 mb

Reply:
We have control instructions. I.e. wasm-gen can produce
```rust
if some_cond {
    while some_cond2 {
        if some_cond3 {
            gr_wait() //with call chance 1/8 or 1/12...?
        }
    }
}
```